APIContract.md

```diff
@@ -0,0 +1,229 @@
+# Yardsourcing API Contract
+
+## Endpoints
+
+| HTTP verbs | Paths  | Used for | Output |
+| ---------- | ------ | -------- | ------:|
+| GET | /api/v1/pets/:type | Get all pets of a given type | [json](#pets-by-type) |
+| GET | /api/v1/users/:user_id | Get a user, and users pets, favorited  pets, and applications| [json](#all-user-data) |
+| POST | /api/v1/users | Create a new user | [json](#create-user) |
+| POST | /api/v1/pets | Create a new pet | [json](#create-pet) |
+| DELETE | /api/v1/pets/:pet_id  | Delete a pet | [details](#delete-a-yard) |
+| PUT | /api/v1/yards/:yard_id | Update a yard | [json](#update-a-yard)
```

Comment:
Change 'yards' here. I'm assuming change it to 'pet'?

Reply:
Thanks I'll fix that.